Vagrantfile

```diff
@@ -6,17 +6,24 @@ NODES = 4
 DISKS = 8
 
 Vagrant.configure("2") do |config|
-    config.vm.box = "chef/centos-7.1"
```

Comment:
Yes. It creates a libvirt-compatible box of the same name. So, we don't
need to override box used based on provider anymore. We still need to
configure the libvirt VMs differently.

On Wed 1 Jul, 2015 9:42 pm Luis Pabón <notifications@github.com> wrote:

> In Vagrantfile
> <https://github.com/heketi/vagrant-heketi/pull/5#discussion_r33694904>:
>
> > @@ -6,17 +6,24 @@ NODES = 4
> >  DISKS = 8
> >
> >  Vagrant.configure("2") do |config|
> > -    config.vm.box = "chef/centos-7.1"
>
> Does the vagrant mutate help with this issue?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/heketi/vagrant-heketi/pull/5/files#r33694904>.
>